js/generate.js

```diff
@@ -25,33 +25,89 @@ function slash(type, p) {
   return {
     'forward': [
       [p[0], p[1], p[3]],
-      [p[1], p[3], p[2]]
+      [p[1], p[3], p[2]],
     ],
     'back': [
+      [p[0], p[2], p[3]],
       [p[0], p[1], p[2]],
-      [p[0], p[2], p[3]]
```

Comment:
Swapped these two polygons to make the concept of `top triangle` and `bottom triangle` work